lib/cert/runner.rb

```diff
@@ -44,6 +41,25 @@ def run
       end
     end
 
+    # Command method for the :revoke_expired sub-command
+    def revoke_expired_certs!
+      login
```

Comment:
Yeah, it doesn't use all params, but I think I agree that it would still be better if a summary table was printed.